src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/entities/camunda/CamundaTask.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.camunda;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import java.time.ZonedDateTime;
+
+import static uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.camunda.CamundaTime.CAMUNDA_DATA_TIME_FORMAT;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+@SuppressWarnings("PMD.TooManyFields")
```

Comment:
Indeed a lot of fields!

Reply:
I think this one is still a WIP, will probably remove a lot of these in the future as we don't need it, but I've also used the lombok data as I don't think it's worth the effort testing this if it's going to be thrown away.

Reply:
yes I think we'll need to think about whether the representation of a Task to client apps needs everything.  Some may be just needed for the internal workings of Task management.